joint1_controller:
  controller:
    package: dynamixel_controllers
    module: joint_position_controller
    type: JointPositionController
  joint_name: joint1
  joint_speed: 1.0
  motor:
    id: 2
    init: 0
    min: 0
    max: 1023